py-polars/polars/internals/lazy_frame.py

```diff
@@ -1071,8 +1070,10 @@ class LazyGroupBy:
     Created by `df.lazy().groupby("foo)"`
     """
 
+    _lgb: "PyLazyGroupBy"
```

Comment:
Renamed from `lgb` to `_lgb`, we dont want this part of the API, do we?

Reply:
Yes, its private. :+1: